packages/transport/test/modules/account/IdentityDeletionProcessController.test.ts

```diff
@@ -0,0 +1,116 @@
+        expect(result).toBeDefined();
+        expect(result.status).toBe(IdentityDeletionProcessStatus.Cancelled);
+    });
+
+    test("should get the active Identity deletion process", async function () {
+        const cancelledIdentityDeletionProcess = await account.identityDeletionProcess.initiateIdentityDeletionProcess();
+        await account.identityDeletionProcess.cancelIdentityDeletionProcess(cancelledIdentityDeletionProcess.id.toString());
+        const activeIdentityDeletionProcess = await account.identityDeletionProcess.initiateIdentityDeletionProcess();
+        const result = await account.identityDeletionProcess.getApprovedIdentityDeletionProcess();
+        expect(activeIdentityDeletionProcess.toBase64()).toBe(result!.toBase64());
+    });
+
+    test("should get all Identity deletion processes", async function () {
+        const cancelledIdentityDeletionProcess = await account.identityDeletionProcess.initiateIdentityDeletionProcess();
+        await account.identityDeletionProcess.cancelIdentityDeletionProcess(cancelledIdentityDeletionProcess.id.toString());
+        const activeIdentityDeletionProcess = await account.identityDeletionProcess.initiateIdentityDeletionProcess();
+        const result = await account.identityDeletionProcess.getIdentityDeletionProcesses();
+        expect(result).toBeDefined();
+        expect(result).toHaveLength(2);
+        expect(result[0].id.toString()).toBe(cancelledIdentityDeletionProcess.id.toString());
+        expect(result[0].status).toBe(IdentityDeletionProcessStatus.Cancelled);
+        expect(result[1].id.toString()).toBe(activeIdentityDeletionProcess.id.toString());
+        expect(result[1].status).toBe(IdentityDeletionProcessStatus.Approved);
+    });
+
+    test("should start an Identity deletion process from the Backbone admin API", async function () {
+        const activeIdentityDeletionProcess = await TestUtil.startIdentityDeletionProcessFromBackboneAdminApi(account);
+        expect(activeIdentityDeletionProcess.status).toBe(IdentityDeletionProcessStatus.WaitingForApproval);
+    });
+
+    test("should not start a second Identity deletion process from the Backbone admin API", async function () {
+        await TestUtil.startIdentityDeletionProcessFromBackboneAdminApi(account);
+        await expect(TestUtil.startIdentityDeletionProcessFromBackboneAdminApi(account)).rejects.toThrow("Request failed with status code 400");
+    });
+
+    test("should not start an Identity deletion process from the Backbone admin API if one was already started by the user", async function () {
+        await account.identityDeletionProcess.initiateIdentityDeletionProcess();
+        await expect(TestUtil.startIdentityDeletionProcessFromBackboneAdminApi(account)).rejects.toThrow("Request failed with status code 400");
+    });
+
+    test("should get the waiting Identity deletion process", async function () {
```

Comment:
Would avoid talking about a "waiting" IdentityDeletionProcess and instead clarify that it is about an Identity deletion process that is "waiting for approval".

Same for the variable name `waitingIdentityDeletionProcess` below.